packages/cli/src/cli/commands/init.test.ts

```diff
@@ -10,7 +10,9 @@ jest.setTimeout(180000);
 const PCC = "./dist/index.js";
 
 beforeAll(async () => {
-  await sh("pnpm run build:staging");
+  // TODO: Need to figure better way of building all packages before running tests
+  await sh("cd ../core && pnpm build");
+  await sh("cd ../cli && pnpm run build:staging");
```

Comment:
Let's solve it the correct way
https://github.com/vercel/turbo/discussions/1347

Reply:
Not building it manually now, rather relying on `pnpm build` step.